react/week1/to-do/src/Todo.js

```diff
@@ -0,0 +1,41 @@
+import React, { Component } from "react";
+
+class Todo extends Component {
+	render() {
+		const { description, deadline } = this.props;
+		return (
+			<li>
+				{description}, {deadline}
+			</li>
+		);
+	}
+}
+
+class Todoes extends Component {
+	render() {
+		const list = [];
+		this.props.todoes.forEach((todo) => {
+			list.push(
+				<Todo description={todo.description} deadline={todo.deadline} />
+			);
+		});
```

Comment:
It would be easier for you if use the `map` function insted, fx:
```suggestion
    const list = this.props.todoes.map(todo => (
      <Todo description={todo.description} deadline={todo.deadline} />
    ));
```